lib/platform/gitlab/index.ts

```diff
@@ -936,21 +936,13 @@ export async function addAssignees(
 ): Promise<void> {
   logger.debug(`Adding assignees '${assignees.join(', ')}' to #${iid}`);
   try {
-    let assigneeId = await getUserID(assignees[0]);
-    let url = `projects/${config.repository}/merge_requests/${iid}?assignee_id=${assigneeId}`;
-    await gitlabApi.putJson(url);
-    try {
-      if (assignees.length > 1) {
-        url = `projects/${config.repository}/merge_requests/${iid}?assignee_ids[]=${assigneeId}`;
-        for (let i = 1; i < assignees.length; i += 1) {
-          assigneeId = await getUserID(assignees[i]);
-          url += `&assignee_ids[]=${assigneeId}`;
-        }
-        await gitlabApi.putJson(url);
-      }
-    } catch (error) {
-      logger.error({ iid, assignees }, 'Failed to add multiple assignees');
+    let url = `projects/${config.repository}/merge_requests/${iid}`;
+    for (let i = 0; i < assignees.length; i++) {
```

Comment:
Can we use url params function from `util/url` to create the parameters instead is this manual creation?

Reply:
Probably needs changes here to handle multi value keys

https://github.com/renovatebot/renovate/blob/85aa03d0dbe453097e11eb02b0b4235cd4207cc9/lib/util/url.ts#L43-L56

Reply:
Didn't know about this one, `getQueryString` already handles keys with multiple values. I will implement this change soon thx